test/testhelpers/assert.go

```diff
@@ -69,3 +70,13 @@ func AssertReportResultsEqual(t *testing.T, expected, actual []map[string]interf
 		}
 	}
 }
+
+func AssertErrorContainsErrorMsgs(t *testing.T, err error, errMsgArr []string) {
+	require.NotNil(t, err, "expected the error would not be nil")
+
+	strErr := strings.ReplaceAll(err.Error(), "\"", "")
```

Comment:
Is this needed?

Reply:
Yes, there's going to be nested `"` that aren't going to be escaped:
```go
failed to install metering: Failed to create the MeteringConfig resource: Failed to create the MeteringConfig resource: MeteringConfig.metering.openshift.io \"operator-metering\" is invalid: []: Invalid value: map[string]interface {}{\"metadata\":map[string]interface {}{\"name\":\"operator-metering\", \"namespace\":\"tflannag-invalidhdfs-missingstoragespec\", \"creationTimestamp\":\"2019-11-05T21:29:17Z\", \"generation\":1, \"uid\":\"525914c9-0013-11ea-abed-0a3e8aca242c\"}, \"spec\":map[string]interface {}{\"logHelmTemplate\":true}, \"kind\":\"MeteringConfig\", \"apiVersion\":\"metering.openshift.io/v1\"}: validation failure list:\nspec.storage in body is required
```

I can add a comment clarifying the intent and describing that behavior.

Reply:
Right, but we aren't using quotes in the errors we're checking for from what I saw, so I wasn't sure if this was actually required or not, since we aren't matching against `"`, it may not matter if they're escaped or not.

Reply:
Hmm I've run this a bunch locally and I'm not hitting any errors after removing the aforementioned.

Previously, I would get something similar to ```... could not be applied builtin len()```, when running the `assert.Contains(...)` so that must have been needed in a prior implementation.